sdk/formrecognizer/Azure.AI.FormRecognizer/src/CopyAuthorization.cs

```diff
@@ -9,47 +9,22 @@ namespace Azure.AI.FormRecognizer.DocumentAnalysis
     [CodeGenModel("CopyAuthorization")]
     public partial class CopyAuthorization
     {
-        /// <summary>
-        /// Initializes a new instance of CopyAuthorization. Used by the <see cref="DocumentAnalysisModelFactory"/>.
-        /// </summary>
-        internal CopyAuthorization(string targetResourceRegion, string targetModelId, string targetModelLocation, DateTimeOffset expiresOn)
-        {
-            TargetResourceRegion = targetResourceRegion;
-            TargetModelId = targetModelId;
-            TargetModelLocation = targetModelLocation;
-            ExpiresOn = expiresOn;
-        }
-
         /// <summary> Initializes a new instance of CopyAuthorization. </summary>
         /// <param name="targetResourceId"> ID of the target Azure resource where the model should be copied to. </param>
         /// <param name="targetResourceRegion"> Location of the target Azure resource where the model should be copied to. </param>
         /// <param name="targetModelId"> Identifier of the target model. </param>
-        /// <param name="targetModelLocation"> URL of the copied model in the target account. </param>
+        /// <param name="targetModelLocation"> URI of the copied model in the target account. </param>
         /// <param name="accessToken"> Token used to authorize the request. </param>
         /// <param name="expiresOn"> Date/time when the access token expires. </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="targetResourceId"/>, <paramref name="targetResourceRegion"/>, <paramref name="targetModelId"/>, <paramref name="targetModelLocation"/>, or <paramref name="accessToken"/> is null. </exception>
-        internal CopyAuthorization(string targetResourceId, string targetResourceRegion, string targetModelId, string targetModelLocation, string accessToken, DateTimeOffset expiresOn)
+        /// <exception cref="ArgumentNullException"> <paramref name="targetResourceId"/>, <paramref name="targetResourceRegion"/>, <paramref name="targetModelId"/>, <paramref name="targetModelLocation"/> or <paramref name="accessToken"/> is null. </exception>
+        /// <exception cref="ArgumentException"> <paramref name="targetResourceId"/>, <paramref name="targetResourceRegion"/>, <paramref name="targetModelId"/> or <paramref name="accessToken"/> is empty. </exception>
+        public CopyAuthorization(string targetResourceId, string targetResourceRegion, string targetModelId, Uri targetModelLocation, string accessToken, DateTimeOffset expiresOn)
```

Comment:
What motivated this change?

Reply:
both Uri to string to making CopyAuthorization publicly initializable?

Reply:
It's just that we always expose URIs as a `Uri` reference in .NET. Architects pointed out we missed `targetModelLocation`.